src/main/java/org/kairosdb/core/tiers/MetricTiersConfigurationUpdateJob.java

```diff
@@ -37,13 +38,13 @@
     public MetricTiersConfigurationUpdateJob(final MetricTiersConfiguration config,
                                              final ObjectMapper objectMapper,
                                              final Executor executor,
-                                             final AccessToken accessToken,
+                                             final AccessTokens accessTokens,
                                              @Named("kairosdb.tiers.schedule") final String schedule,
                                              @Named("zmon.hostname") final String hostname) {
         this.config = config;
         this.objectMapper = objectMapper;
         this.executor = executor;
-        this.accessToken = accessToken;
+        this.accessToken = accessTokens.getAccessToken("zmon-read");
```

Comment:
why is the token name mentioned twice?

Reply:
because `AccessTokens` object can manage different tokens. That's not needed for this use case, but that's how SDK work